src/components/badge/badge.stories.mdx

```diff
@@ -63,11 +64,12 @@ Avoid using badges for critical links: Their appearance is too subtle to rely on
 - `class` (string)
 - `href` (string, optional): A URL for the badge to link to.
 - `icon` (string, optional): The name of one of [our icons](/docs/design-icons--page) to display.
-- `message` (string, default `'Label'`)
+- `label` (string, default `'Label'`)
```

Comment:
This appears to be a preexisting typo, there is no `message` property in this template.